entgql/annotation.go

```diff
@@ -86,6 +93,21 @@ func Skip() Annotation {
 	return Annotation{Skip: true}
 }
 
+// RelayConnection returns a relay connection annotation.
+func RelayConnection() Annotation {
+	return Annotation{RelayConnection: true}
+}
+
+// Implemented returns an Implemented annotation.
+func Implemented(interfaces ...string) Annotation {
```

Comment:
I think it should be named as `Implements`, because it's from the implementor perspective. A `User` implements `Node` (as in GraphQL).